tools/torch/data.lua

```diff
@@ -489,34 +674,33 @@ function DBSource:totalRecords ()
 end
 
 -- Derived class method close (LMDB flavour)
-function DBSource:lmdb_close (batchsize)
+function DBSource:lmdb_close (batchSize)
```

Comment:
Shall I remove it while I'm at it? Note it's also in the other db closure `DBSource:hdf5_close (batchSize)`